mesa/experimental/cell_space/cell_collection.py

```diff
@@ -110,6 +110,41 @@ def select_random_agent(self) -> CellAgent:
         """
         return self.random.choice(list(self.agents))
 
+    def sample_random_cells(self, k: int = 1, counts: list | None = None) -> list[Cell]:
+        """Sample k unique cells from the cell collection.
+
+        See random.sample for more information.
```

Comment:
Trying to improve the API would it be better to be more explicit here 

`See Python's random.sample at python.docs.org for more information`